soh/soh/SaveManager.cpp

```diff
@@ -1247,13 +1436,9 @@ void CopyV0Save(SaveContext_v0& src, SaveContext& dst) {
     }
     dst.worldMapAreaData = src.worldMapAreaData;
     dst.scarecrowCustomSongSet = src.scarecrowCustomSongSet;
-    for (size_t i = 0; i < ARRAY_COUNT(src.scarecrowCustomSong); i++) {
-        dst.scarecrowCustomSong[i] = src.scarecrowCustomSong[i];
-    }
+    memcpy(&dst.scarecrowCustomSong[0], &src.scarecrowCustomSong[0], sizeof(src.scarecrowCustomSong));
```

Comment:
does struct assignment not work for this like it did previously?

Reply:
`dst` is a `SaveContext`, and `scr` a `SaveContext_v0` where the scarecrow song is still a u8 array.
So the u8 array needs to be converted to OcarinaNotes, where the simplest way is just copying all bytes into the struct.